src/common/generator.js

```diff
@@ -533,7 +533,7 @@ window.POG=(function() {
         removeNodes(hiddens);
         // ng:view template doesn't have height,
         // hence it will considered as hidden
-        if (clonedNode.textContent.trim() === '') {
+        if (typeof clonedNode.textContent === 'string' && clonedNode.textContent.trim() === '') {
```

Comment:
We would want to exclude the node when textContent is null in conjuction of empty string. It might be better to use default value on textContent null.

if ((clonedNode.textContent || '').trim() === '') {

Reply:
makes sense. apologies for my shortsightedness. i've sent in a new commit